packages/api/src/controllers/asset.ts

```diff
@@ -287,6 +287,24 @@ async function validateAssetPlaybackPolicy(
         `webhook ${playbackPolicy.webhookId} not found`
       );
     }
+    const allowedOrigins = playbackPolicy?.allowedOrigins;
+    if (allowedOrigins) {
+      if (allowedOrigins.length > 0) {
+        const allowedOriginsValid = allowedOrigins.every((origin) => {
+          const url = new URL(origin);
+          return (
+            ["http:", "https:"].includes(url.protocol) &&
+            url.hostname &&
+            url.port
```

Comment:
Not sure if necessary (or even correct), but should we check that there is no path or the path is just `/`? I'm not sure if there could be a path tho, gotta read spec

Reply:
They should never put the path, even with the final slash, I will add a check